test/test_rest_vol.c

```diff
@@ -1473,8 +1473,6 @@ test_unused_file_API_calls(void)
 
         if (H5Fflush(file_id, H5F_SCOPE_GLOBAL) >= 0)
             TEST_ERROR
-        if (H5Fget_obj_count(file_id, H5F_OBJ_DATASET) >= 0)
```

Comment:
This is supported now, so it's no longer expected to fail. Actual tests for it are in `h5vl_test`.